bench/telemetry/run.sh

```
@@ -0,0 +1,64 @@
+CONC=${CONC:-64}
+AWAITS=${AWAITS:-2}
+REPS=${REPS:-3}
+SERVER_CPU=${SERVER_CPU:-4}
+LOAD_CPUS=${LOAD_CPUS:-8-23}
+here=$(cd "$(dirname "$0")" && pwd)
+PORT=$((20000 + RANDOM % 20000))
+COLLECTOR_PORT=$((PORT + 1))
+TCK=$(getconf CLK_TCK)
+out=$(mktemp -d)
+
+COLLECTOR_PORT=$COLLECTOR_PORT taskset -c 30 $BUN "$here/collector.js" 2>/dev/null & COL=$!
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)